pkg/skaffold/schema/v2beta6/upgrade.go

```diff
@@ -24,6 +24,8 @@ import (
 
 // Upgrade upgrades a configuration to the next version.
 // Config changes from v2beta6 to v2beta7
+// 1. Additions:
+//		New KptDeploy, KptFn, and KptLive to support Kpt Deployer.
```

Comment:
nit: remove the second ","?

Reply:
Not if you like oxford commas :-)

Reply:
to be honest, I didn't know this before. Thanks Brian and @felixtran39 😄